pennylane/gradients/pulse_gradient.py

```diff
@@ -188,9 +225,13 @@ def _stoch_pulse_grad(tape, argnum=None, num_split_times=1, sampler_seed=None, s
             rule underlying the differentiation; also see details
         sample_seed (int): randomness seed to be used for the time samples in the stochastic
             parameter-shift rule
-        shots (None, int, list[int]): The device shots that will be used to execute the tapes outputted by this
-            transform. Note that this argument doesn't influence the shots used for tape execution, but provides
-            information about the shots.
+        shots (None, int, list[int]): The device shots that will be used to execute the tapes
+            outputted by this transform. Note that this argument doesn't influence the shots
+            used for tape execution, but provides information about the shots.
```

Comment:
> Note that this argument doesn't influence the shots used for tape execution, but provides information about the shots.

I dont get this sentence 🤔 Is there something missing? / can you rewrite it please

Reply:
This is copied from some other transform :sweat_smile: 
I tried rewriting it a bit, let me know what you think :)

Reply:
Continuing here https://github.com/PennyLaneAI/pennylane/pull/4004/files#r1169819951